axelrod/strategies/finite_state_machines.py

```diff
@@ -101,14 +123,15 @@ class Fortress3(FSMPlayer):
     }
 
     def __init__(self) -> None:
-        transitions = (
-            (1, D, 2, D),
+        transitions = [
```

Comment:
mypy gave me trouble on the change.  it only worked it the type hints were a generalized tuple. it did have the nice advantage that i could set the defaults in the kwargs and not after the fact.